backend/src/main/java/com/zeros/backend/models/RestaurantAffiliate.java

```diff
@@ -0,0 +1,37 @@
+package com.zeros.backend.models;
+
+import java.time.LocalDateTime;
+
+public class RestaurantAffiliate extends Entity {
+
+    private final Long restaurantId;
+    private LocalDateTime affiliateDate;
+    private final Long qrCode;
+
+
+    public RestaurantAffiliate(Long restaurantId, Long qrCode, LocalDateTime affiliateDate) {
+        this.restaurantId = restaurantId;
+        this.qrCode = qrCode;
+        this.affiliateDate = affiliateDate;
+    }
+
+    public Long getRestaurantId() {
+        return restaurantId;
+    }
+
+    public LocalDateTime getAffiliateDate() {
+        return affiliateDate;
+    }
+
+    public Long getQrCode() {
+        return qrCode;
+    }
+
+    public boolean isSigned() {
+        return affiliateDate.isEqual(LocalDateTime.now()) || affiliateDate.isBefore(LocalDateTime.now());
+    }
```

Comment:
i don't understand the meaning of function isSigned :(

Reply:
The restaurant shoud decide when hi will start donation he can accepte but next month or next year.

Reply:
When an accessors returns the value of a property that represents and action, I often use the prefex 'has' instead of 'is', we can do the same, in this case the result is 'hasSigned'

Reply:
When an accessors returns the value of a property that represents and action, I often use the prefex 'has' instead of 'is', we can do the same, in this case the result is 'hasSigned'

Reply:
When The property is a boolean we say the convention is signed i think

Reply:
I think that both 'is' and 'has' can be used for Booleans depending on the signification of the property.